lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -117,6 +117,14 @@ sub ensure_size {
     return $size;
 }
 
+sub hidden {
+    # 1 if asset should not be linked from the web UI (as set by
+    # 'hide_asset_types' config setting), otherwise 0
+    my ($self) = @_;
+    my @types = split(/ /, $app->config->{global}->{hide_asset_types});
```

Comment:
$OpenQA::Utils::app for models to access the app - this should also fix the tests